js/components/charts/frequencyHistogram.js

```diff
@@ -2,14 +2,20 @@ define([
 	'knockout',
   'pages/data-sources/const',
   'providers/Chart',
+  'providers/Component',
```

Comment:
Apply proper formatting (as I remember, somewhere OHDSI decided to use tabs, but it's better to double-check). It is impossible to read code now